web/src/lib/components/pages/faq.tsx

```diff
@@ -30,6 +30,10 @@ export default class FAQ extends Component<Props, State> {
 
       <h3>Why is 10,000 hours the goal for capturing audio?</h3>
       <p>This is approximately the number of hours required to train a production STT system.</p>
+
+      <h3>Where can I find the sample text?</h3>
+      <p>Common Voice is an open source project that is on <a href="https://github.com/mozilla/voice-web/">Github</a></p>
+      <p>Our sets of samples are available <a href="https://github.com/mozilla/voice-web/blob/master/server/data/temporary-sentences.txt">here</a>, <a href="https://github.com/mozilla/voice-web/blob/master/server/data/temporary-sentences-2.txt">here</a>, <a href="https://github.com/mozilla/voice-web/blob/master/server/data/funny.txt">here</a>, and <a href="https://github.com/mozilla/voice-web/blob/master/server/data/new-sentences.txt">here</a>.</p>
```

Comment:
Right now, we just use `https://github.com/mozilla/voice-web/blob/master/server/data/new-sentences.txt`.